.circleci/config.yml

```diff
@@ -560,11 +572,24 @@ jobs:
                 command: ./bin/build version
 
   fe-tests-cypress:
-    executor: clojure-and-node
+    parameters:
+      e:
+        type: executor
+        default: clojure-and-node
+      only-single-database:
+        type: boolean
+        default: false
+      test-files-location:
+        type: string
+        default: ""
+      driver:
```

Comment:
Where is this used?

Reply:
It’s used as part of the `run_cypress_tests` below. So there’s a Boolean flag to enable the database specific tests and then this passes in a path that those tests live in. Currently this is only `frontend/test/metabase-db/mongo` but will eventually be other DB types.